tools/psa/__init__.py

```diff
@@ -20,9 +20,6 @@
 
 from tools.resources import FileType
 from tools.settings import ROOT
-from .generate_partition_code import manifests_discovery, generate_spm_code
-
-
 
 def find_secure_image(notify, resources, ns_image_path,
```

Comment:
Who uses `find_secure_image()` still? I am curious if we need any files in `tools/psa` at all.

Reply:
@Patater Currently, `find_secure_image()` is not needed. Having said that, once we add tf-m binaries to targets then we need this method to find the secure image and merge it with non-secure image. It's better to retain it.
The python scripts under `tools/psa/tfm` are needed to sign the combined binary, but they are not up to date. I can delete the folder now and bring those files back when we import the NS APIs from tf-m using importer script or retain the folder. Any recommendation?
I'll delete `partition_description_schema.json` and `README.md`.

Reply:
@Patater any recommendation?

Reply:
@jainvikas8 Mentioned that this would be needed for any v8-M targets, which we don't care to support from this branch.

Please correct me if I'm wrong, but I don't believe we'll need it to build the combined image if we place the binary in the proper location in the Mbed OS tree. No special "find" step should be needed as the globbing build system will pull in everything it can. I'm thinking the SPE binaries would go in a similar location (similar filter folders) to the linker script for each target.

Reply:
@Patater Got it. Removed `tools/psa` folder.